UI/assets/css/auth.css

```diff
@@ -0,0 +1,8 @@
+body{
+    background-image: url("../images/fast_foods.jpg");
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-attachment: fixed; 
+    font-family: Tahoma, sans-serif;
+}
```

Comment:
add a new line at the end of this file and everywhere else you see the red marking